src/ElmSharp/ElmSharp/EvasObject.cs

```diff
@@ -125,6 +125,8 @@ public abstract class EvasObject
         private Interop.Eext.EextEventCallback _backButtonHandler;
         private Interop.Eext.EextEventCallback _moreButtonHandler;
 
+        private static Dictionary<IntPtr, EvasObject> _objects = new Dictionary<IntPtr, EvasObject>();
```

Comment:
There is no adding item block in this code.

Reply:
Oh... I miss some piece of code in merging.

Reply:
I add missed code in line 1083

Reply:
The static variable must be prefixed with s_ ...
and I guess the name is not good.
Would not it be nice to have an intuitive name like s_HandleToItemTable in an ItemObject?